.ci/Jenkinsfile

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu && immutable' }
+  environment {
+    BASE_DIR="src/github.com/elastic/elastic-agent-client"
+    PIPELINE_LOG_LEVEL='INFO'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
```

Comment:
This is a very large timeout for a simple client, I would hate for it to get stuck for some reason and it just sit for 1 hour. I think like 15 mins or 30 mins at the most.

Reply:
It is a default value, this timeout is only used when something goes really wrong and the build stucks for some reason. We usually adjust this value to a more proper value after a few build on the CI.